taskvine/src/manager/taskvine.h

```diff
@@ -38,6 +38,14 @@ expected events.
 #define VINE_RANDOM_PORT  0                  /**< Indicates that any port may be chosen. */
 #define VINE_WAITFORTASK  -1                 /**< Timeout value to wait for a task to complete before returning. */
 
+/** **/
```

Comment:
Yes, this is the right idea, but it is an implementation detail and not part of the public API, so don't put it in `taskvine.h`, just put it at the top of `vine_file.c`

Reply:
moved.